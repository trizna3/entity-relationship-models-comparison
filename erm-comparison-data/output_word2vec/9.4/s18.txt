line-lines
vehicle type-vehicle type
stop-stops
vehicle-EMPTY
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-vehicle type-lines
MOVE_ATTR_TO_INCIDENT_ASSOCIATION:S-order-(stops(ONE),routes(MANY))
REBIND_1NN1_TO_MN:S-routes
REBIND_1NN1_TO_MN:S-schedule
