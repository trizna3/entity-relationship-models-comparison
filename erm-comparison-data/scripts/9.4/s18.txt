entity set stops {
	"name"
};

entity set lines {
	"number"
	"vehicle type"
};

entity set vehicles {
	"code"
	"timestamps"
};

entity set routes {
	"order"
	"duration"
};

entity set schedule {
	"time"
};

entity set busses {
};

entity set trolleybusses {
};

entity set trams {
};

association between {
	stops ONE
	routes MANY
}{};

association between {
	lines ONE
	routes MANY
}{};

association between {
	lines ONE
	schedule MANY
}{};

association between {
	schedule MANY
	vehicles ONE
}{};

generalization from busses to vehicles;

generalization from trolleybusses to vehicles;

generalization from trams to vehicles;