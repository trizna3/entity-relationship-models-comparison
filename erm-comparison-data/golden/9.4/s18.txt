vehicle-vehicles
vehicle type-EMPTY
line-lines
stop-stops
GENERALIZATION_TO_11_ASSOCIATION:S-busses-vehicle
GENERALIZATION_TO_11_ASSOCIATION:S-trolleybusses-vehicle
GENERALIZATION_TO_11_ASSOCIATION:S-tram-vehicle
CONTRACT_11_ASSOCIATION:S-busses-vehicle
CONTRACT_11_ASSOCIATION:S-trolleybusses-vehicle
CONTRACT_11_ASSOCIATION:S-tram-vehicle